Description: replace YouTube iframe with YouTube link
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: not-needed
Last-Update: 2022-09-11

--- a/doc/src/getting-started/gettingstarted.qdoc
+++ b/doc/src/getting-started/gettingstarted.qdoc
@@ -79,10 +79,8 @@
     To complete the installation, select the components that you want to install
     and follow the instructions of the installation program.
 
-    The following YouTube video gives a complete walk-through of the
-    installation process:
-
-    \youtube 232cwcW5atA
+    The \l{https://www.youtube.com/watch?v=232cwcW5atA}{following YouTube video}
+    gives a complete walk-through of the installation process.
 
     \section2 Updating or Removing Qt
 
